src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -158,13 +158,30 @@ export class PatientViewPageStore {
         return this._sampleId ? 'sample' : 'patient';
     }
 
-    @computed get mutationGeneticProfileId() {
-        return `${this.studyId}${GENETIC_PROFILE_MUTATIONS_SUFFIX}`;
-    }
+    readonly mutationGeneticProfileId = remoteData({
+        await: () => [
+            this.geneticProfilesInStudy
+        ],
+        invoke: async() => {
+            const profile = this.geneticProfilesInStudy.result.find((p: GeneticProfile) => {

```

Comment:
We should probably move this invoke function into `StoreUtils` to reuse within `ResultsPageViewStore` (after merging #330).